src/opencensus/proto/tags/v1/tags.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package opencensus.proto.tags.v1;
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/tags/v1";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.tags.v1";
+option java_outer_classname = "TagsProto";
+
+// TagScope is used to determine the scope of a Tag.
+message TagScope {
+  enum Scope {
```

Comment:
Is this information useful for exporter?

Reply:
My understanding - this information is only needed for API to decide what to do with the tag

Reply:
Correct, this is not needed for the exporter, but I defined it here for few reasons:
1. To have the language independent definition of the data model.
2. In case anyone wants to use a static config (json file to define tags that are set).

Reply:
Do you want exporters to set the `scope` value? If it is no use in agent and collector - there should be a comment saying that this field is only needed for data generation and tags defining cases, not for exporting?

Reply:
done.